### PR TITLE
[Polygon] Bor: Added Ahmedabad HF related configs and block number for mainnet network

### PR DESCRIPTION
**Pull request by pratikspatil024**
[Polygon] Bor: Added Ahmedabad HF related configs and block number for mainnet network
This is the Ahmedabad block number - [62278656](https://polygonscan.com/block/countdown/62278656)

PR in bor - [bor#1330](https://github.com/maticnetwork/bor/pull/1330)
PR for erigon-2 - [erigon#11979](https://github.com/erigontech/erigon/pull/11979)

### CODE REVIEW

**Comment by pratikspatil024** [comment]
Please merge this only after the bor PR is merged, thanks!

**Review by marcello33 — APPROVED** [review]

**Review by dev1644 — COMMENTED** [review]
LGTM!

**Review by DhairyaSethi — APPROVED** [review]

**Comment by pratikspatil024** [comment]
We are good to merge. A stable tag is created in bor ([v1.4.0](https://github.com/maticnetwork/bor/releases/tag/v1.4.0)).

**Review by taratorio — APPROVED** [review]

**Pull request merged**
